src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The truck stopped working");

```

Comment:
The message 'The truck stopped working' should be more informative and indicate the type of Machine. For example, 'Truck has stopped its work'.